src/locales/en/common.json

```diff
@@ -1923,7 +1923,7 @@
           "receiveFloat": "Amount to receive before service fees",
           "provider": "Provider",
           "method": "Rate",
-          "fees": "Max fees",
+          "fees": "Fees",
```

Comment:
This is exactly what I meant, by changing wording here, you are altering the v1 text too and that shouldn't happen at all. Please sync with @LFBarreto in case he wants to do things differently but to me we need a swapv2 namespace in the wording file that has nothing to do with the v1

Reply:
I changed it, as it was not used anywhere else.
But you are right, it's not a good thing, if we need to keep the 2 swap in the codebase, we should have a different namespace.
I will create a swapv2 namespace